blueocean-github-pipeline/src/main/java/io/jenkins/blueocean/blueocean_github_pipeline/GithubServerContainer.java

```diff
@@ -97,8 +100,12 @@
                             inputStream = HttpRequest.getErrorStream(connection);
                         }
 
+                        if (inputStream == null) {
+                            throw new IOException();
+                        }
```

Comment:
Any particular reason for this? It looks kind of unrelated to the other changes. If it occurs in practice it would be nice to an error message.

Reply:
The tests expect the `readValue()` to throw an `IOException` when `inputStream` is `null`.  In the current version of Jackson, it now throws `IllegalArgumentException` producing a different result. 

https://github.com/jenkinsci/blueocean-plugin/pull/2064/files#diff-83cc03e1fffef92465efa766f12eb5f6R117

The right thing is probably to catch `IllegalArgumentException` instead.  I'll fix this up.